docs/data/charts/composition/composition.md

```diff
@@ -71,7 +72,7 @@ It can be used instead of them when there is no need to customize anything outsi
 ### Graphical components
 
 These are any component that render the graphical elements of the chart.
-They are the children of the **Structural components** shown above.
+They are the children of the **SrtsDataProvider, and thus a customized legend. Before, in v7, it was possible to use ChatsLegend inside ChartContainer with together with multiple Line/BarPlots, unfortunately this page as no indication/help/example for that scenario. All graphs rendered here hatructural components** shown above.
```

Comment:
Does this sentence make sense here? 

If we're referring to something from v7, should we use a `:::info`?

Reply:
I think it's a bad copy paste. I don't know why but those time VSCode struggles with copy paste. It's lagging, so not impossible I missed a wong one